From c0a01e6aa91cab7ba977ffb019a171f7d91f60d9 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Tue, 22 Sep 2020 11:41:58 -0500
Subject: [PATCH 26/44] arm64: dts: beacon-renesom-baseboard: Fix Audio
 clocking on beta hardware

The audio on alpha hardware did not support both 44.1 and 48KHz clocks.
This functionality was added in beta hardware, which required some
changing of how the clocks were wired.

This patch fixes the audio clocking on beta hardware to support both.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)
 mode change 100644 => 100755 arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi

diff --git a/arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi b/arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi
old mode 100644
new mode 100755
index f8e74eca7a4e..f68c17385de3
--- a/arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi
+++ b/arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi
@@ -292,7 +292,7 @@
 	status = "okay";
 	/* Hack, we are asking the EHCI to take a clk ref to the Ethernet Phy
 	refclock so Ethernet will work */
-	clocks = <&cpg CPG_MOD 703>, <&cpg CPG_MOD 704>, <&versaclock5 4>;
+	clocks = <&cpg CPG_MOD 703>, <&cpg CPG_MOD 704>, <&versaclock5 4>, <&versaclock6_bb 3>;
 };
 
 &hdmi0 {
@@ -413,7 +413,7 @@
 
 		OUT3 {
 			idt,mode = <VC5_CMOS>;
-			idt,voltage-microvolts = <1800000>;
+			idt,voltage-microvolts = <3300000>;
 			idt,slew-percent = <100>;
 		};
 
@@ -620,7 +620,7 @@
 	};
 
 	sound_clk_pins: sound_clk {
-		groups = "audio_clk_a_a";
+		groups = "audio_clk_a_a", "audio_clk_b_a";
 		function = "audio_clk";
 	};
 
-- 
2.17.1

